8-objects/task_2.js

```diff
@@ -0,0 +1,59 @@
+const characters = [
+	{ 'name': 'Barney', 'age': 36 },
+	{ 'name': 'Fred', 'age': 40 },
+	{ 'name': 'Jack', 'age': 50 }
+];
+
+// 1. Напишите функцию addCharacter(character) позволяющую добавить новый объект в массив characters. 
+// Объект должен иметь поля name (string) и age (number)
+
+function addCharacter(name, age) {
+	const newObject = {};
+	newObject.name = name;
+	newObject.age = age;
+	return characters.push(newObject);
```

Comment:
```suggestion
	const newObject = { name, age };
	characters.push(newObject);
```
Зачем тебе ретурнать результат пуша? ты вернешь длину нового массива, и где ты ее применишь?:)

Reply:
Если хочешь ретурнать что-то - возвращай массив characters